script.js

```diff
@@ -283,6 +283,9 @@ const cardMarkup = movie => `
   <h5>${movie.director}</h5>
   <h5>${movie.movie_year}</h5>
 </div>
+`;
+
+const ratingMarkup = movie => `
```

Comment:
It looks like you declare this function but never call it?

This is where the `movie-rating` element in your code you want to grab but it isn't ever created